### DISCUSSION TITLE
Unable to update templates to latest Version

### DISCUSSION THREAD

**Discussion by nmanda89** [projectdiscovery/nuclei-templates#10618] [Q&A]
Hi. I am trying to install nuclei and scan my project URL  but after installation i am getting 
![image](https://github.com/user-attachments/assets/8af05540-5b8d-4a92-8280-646abc4d9ab7)

**Comment by GeorginaReeder** [comment]
Hi there @nmanda89 ! Just to check here, what's the method/command you're using to install nuclei here?

Tagging @dwisiswant0 in the meantime.

**Reply by nmanda89** [reply]
go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest
this is the command i used to install nuclei and i can get the nuclei version with command nuclei -version

**Comment by dwisiswant0** [comment]
Can you rerun the command with the addition of the `-verbose` flag and see what the output looks like, @nmanda89?

And before you do that, it appears your templates are outdated. Try updating to the latest version using the `-update-templates` flag.

**Reply by nmanda89** [reply]
![image](https://github.com/user-attachments/assets/4892337e-629f-4685-817c-08cb92ce5b42)
This is the command output i got @dwisiswant0

**Reply by dwisiswant0** [reply]
```bash
go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest
nuclei -version
nuclei -update-templates
nuclei -u URL
```

**Reply by nmanda89** [reply]
Same issue again.
![image](https://github.com/user-attachments/assets/e9e623d4-2c81-4310-8845-e4e079e663fb)

**Reply by nmanda89** [reply]
Can someone resolve my issue please. [GeorginaReeder](https://github.com/GeorginaReeder)

**Reply by GeorginaReeder** [reply]
Hi @nmanda89 , we are looking into this. @dwisiswant0 , is there anything else that can be done here?

**Reply by dwisiswant0** [reply]
I'm not so sure about this. Would you mind attaching a complete screenshot of the results from the following commands that I've recommended previously for clearer understanding?

> ```shell
> go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest
> nuclei -version
> nuclei -update-templates
> nuclei -u URL -v
> ```

**Reply by nmanda89** [reply]
Hi @dwisiswant0 . I have run all the commands that you recommended and below is the results
![image](https://github.com/user-attachments/assets/37af7862-3a85-438e-b534-b01390043f22)
